.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
```

Comment:
I like this set of templates, with the exception of the bug report. In this stage in the product's journey, I think we want to make it as low friction as possible to submit a bug report. We don't have an issue spam issue, and we're honestly desperate to get bug reports as quickly as possible.

How about we make bug either entirely free-form or simplify it to two questions, e.g. (What happened?) (What did you expect would happen?).

Reply:
> How about we make bug either entirely free-form or simplify it to two questions, e.g. (What happened?) (What did you expect would happen?).

👍 Done!

>  In this stage in the product's journey. I think we want to make it as low friction as possible to submit a bug report. 

Agreed!